README.md

```diff
@@ -1,16 +1,39 @@
+## Be safe with your crypto assets!
+
+This application is a community maintained project.
+We at NuCypher have every indication that this site was created by good people,
+but we cannot feasibly ensure that the code running here and all
+its dependencies are absolutely 100% safe.
+
+As always - when interacting with web-based blockchain
+applications, you the user are responsible for confirming
+that your wallet and / or client are interacting with the correct contract.
+
+Below are the only valid addresses for interacting with NuCypher at this time.
+If this application attempts to interact with any other address, please drop what
+you're doing and report it to security@nucypher.com and to our Discord.
```

Comment:
This says the addresses are below but I don't see them.

Reply:
:heavy_check_mark: